<a href="https://colab.research.google.com/github/cla0r/exe/blob/main/Avalia%C3%A7%C3%A3o_ALGOR%C3%8DTIMOS.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

QUESTÃO 01

Faça um Programa Python que converta metros para centímetros.

In [9]:
metros = float(input("Metros = "))
centimetros = metros * 100
print("cm = ",centimetros)

Metros = 5
cm =  500.0


QUESTÃO 02

Faça um Programa Python que pergunte quanto você ganha por hora e o número de
horas trabalhadas no mês. Calcule e mostre o total do seu salário no referido mês.

In [11]:
ganhoPhora = float(input("Ganho por hora = R$ "))
horasTrabalhadas = float(input("Horas trabalhadas ao mês = "))
salarioMensal = ganhoPhora * horasTrabalhadas
print("Salário mensal = R$", salarioMensal)

Ganho por hora = R$ 25
Horas trabalhadas ao mês = 220
Salário mensal = R$ 5500.0


QUESTÃO 03

Faça um Programa Python que peça a temperatura em graus Farenheit, transforme e
mostre a temperatura em graus Celsius. C = (5 * (F-32) / 9).

In [ ]:
tempFarenheit = float(input("F = "))
tempCelcius = 5 * (tempFarenheit - 32) / 9
print("Temperatura em Celcius= ", tempCelcius)

F = 98
Temperatura em Celcius=  36.666666666666664


QUESTÃO 04

Uma loja está vendendo seus produtos em 5 (cinco) prestações sem juros. Faça um
programa Python que receba um valor de uma compra e mostre o valor das
prestações.

In [12]:
valorCompra = float(input("Valor da Compra = R$ "))
valorPrest = valorCompra / 5
print("Valor da Prestação = R$", valorPrest)

Valor da Compra = R$ 1350.47
Valor da Prestação = R$ 270.094


QUESTÃO 05

Faça um programa Python que receba o preço de custo de um produto e mostre o
valor de venda. Sabe-se que o preço de custo receberá um acréscimo de acordo com
um percentual informado pelo usuário.

In [13]:
preco = float(input("Preço de custo = R$ "))
porcentagem = preco * 13/100
valorVenda = preco + porcentagem
print("Valor da Venda = R$", valorVenda)

Preço de custo = R$ 400
Valor da Venda = R$ 452.0


QUESTÃO 06

Faça um programa Python que receba o preço de custo de um produto e mostre o
valor de venda. Sabe-se que o preço de custo receberá um acréscimo de acordo com
um percentual informado pelo usuário.

In [ ]:
distanciaP = float(input("km =  "))
combustivelGasto = float(input("l = "))
consumoM = distanciaP / combustivelGasto
print("Consumo Mensal= ", consumoM)


km =  30
l = 2
Consumo Mensal=  15.0


QUESTÃO 07

Faça um programa Python que leia três números inteiros e positivos (A, B e C) e calcule a seguinte expressão:

D=R+S/2  onde, R=(A+B)**2    S=(B+C)**2


In [ ]:
a = int(input("A= "))
b = int(input("B= "))
c = int(input("C= "))
R = (a + b) **2
S = (b + c) **2
D = (R + S) /2
print("R= ",R)
print("S= ",S)
print("D= ",D)

A= 2
B= 4
C= 8
R=  36
S=  144
D=  90.0


QUESTÃO 08

Escrever um programa Python que leia o nome de um vendedor, o seu salário fixo e o total de vendas efetuadas por ele no mês (em dinheiro). Sabendo que este vendedor ganha 15% de comissão sobre suas vendas efetuadas, informar o seu nome, o salário fixo e salário no final do mês.

In [ ]:
nome = str(input("Nome do Vendedor: "))
salarioFixo = float(input("Salário Fixo em R$: "))
totalVendas = int(input("Total de Vendas: "))
porcentagem = salarioFixo * 0.15
salarioFinal = salarioFixo + porcentagem
print("Salário Final = R$ ", salarioFinal)

Nome do Vendedor: Astrogildo
Salário Fixo em R$: 1212
Total de Vendas: 20
Salário Final = R$  1393.8


QUESTÃO 09

Faça um Programa Python que peça 2 números inteiros e um número real. Calcule e mostre: 
A) o produto do dobro do primeiro com metade do segundo . 
B) soma do triplo do primeiro com o terceiro.     
C) o terceiro elevado ao cubo. 


In [ ]:
num1 = int(input("N1: "))
num2 = int(input("N2: "))
num3 = float(input("N3: "))
produtoA = (num1 * 2) * (num2 / 2)
somaB = (num1 * 3) + num3
aoCuboC = num3 ** 3
print("A = ", produtoA)
print("B = ", somaB)
print("C = ", aoCuboC)

N1: 4
N2: 5
N3: 7
A =  20.0
B =  19.0
C =  343.0


QUESTÃO 10

Tendo como dados de entrada a altura de uma pessoa, construa um programa Python que calcule seu peso ideal, usando a seguinte fórmula: (72.7*altura) – 58.


In [ ]:
altura = float(input("h = "))
pesoIdeal = (72.7*altura) - 58
print("Peso ideal = ", pesoIdeal)

h = 1.60
Peso ideal =  58.32000000000001


QUESTÃO 11

Faça um programa Python que peça o tamanho de um arquivo para download (em MB) e a velocidade de um link de Internet (em MBps), calcule e informe o tempo aproximado de download do arquivo usando este link (em minutos).

In [ ]:
tamArquivo = float(input("Tamanho do arquivo em MB:"))
velArquivo = float(input("Velocidade do arquivo em MBps:"))
segPmin = velArquivo / 60
tempoDownload = tamArquivo / segPmin
print("Tempo em min =", tempoDownload)


Tamanho do arquivo em MB:200
Velocidade do arquivo em MBps:100
Tempo em min = 120.0


QUESTÃO 12

Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas no mês. Calcule e mostre o total do seu salário no referido mês, sabendo-se que são descontados 11% para o Imposto de Renda, 8% para o INSS e 5% para o sindicato, faça um programa Python que nos dê: 
a) salário bruto. 
b) quanto pagou ao INSS. 
c) quanto pagou ao sindicato. 
d) o salário líquido. 
e) calcule os descontos e o salário líquido, conforme a tabela abaixo:
Salário Bruto : R$
IR (11%) : R$
INSS (8%) : R$
Sindicato ( 5%) : R$
Salário Liquido : R$
Obs.: Salário Bruto - Descontos = Salário Líquido.


In [ ]:
ganhoPhora = float(input("Ganho por hora = R$ "))
horasTrabalhadas = float(input("Horas trabalhadas ao mês = "))
salarioBruto = ganhoPhora * horasTrabalhadas
print(salarioBruto)
ir = (salarioBruto* 0.11)
inss = (salarioBruto * 0.08)
sind = (salarioBruto  * 0.05)
salarioLiquido = salarioBruto - (ir + inss + sind)
descontos = ir + inss + sind
print("Salario Liquido = ", salarioLiquido)
print("inss = ", inss)
print("sind = ", sind)
print("descontos =", descontos)

Ganho por hora = R$ 25
Horas trabalhadas ao mês = 220
5500.0
Salario Liquido =  4180.0
inss =  440.0
sind =  275.0
descontos = 1320.0


QUESTÃO 13

Escrever um programa em Python que lê o público total de um jogo de futebol e fornecer a renda do jogo, sabendo-se que havia 4 tipos de ingressos assim distribuídos: popular 10% a R$10,00, geral 50% a R$50,00, arquibancada 30% a R$100,00 e cadeiras 10% a R$ 200,00.

In [ ]:
publico = int(input("Público total do jogo: "))
popular = (publico * 0.10) * 10 # (10/100=0.10)
geral = (publico * 0.50) * 50
arquibancada = (publico * 0.30) * 100
cadeiras = (publico * 0.10) * 200
renda = popular + geral + arquibancada + cadeiras

print('A renda desse jogo de futebol foi de: R$ ', renda)

Público total do jogo: 500
A renda desse jogo de futebol foi de: R$  38000.0


QUESTÃO 14

Na fazenda de João são criadas aves. Para alimentá-las ele compra sacas de ração cujo peso é fornecido em quilos. No total ele tem dois grupos de aves, para os quais fornece a quantidade de ração em gramas. A quantidade diária de ração fornecida para cada grupo de aves é sempre a mesma. Faça um programa em Python que receba o peso do saco de ração e a quantidade de ração fornecida para cada grupo de aves, calcule e mostre quanto restará de ração após uma semana.

In [ ]:
PsacoG1 = float(input("Peso do saco de ração Grupo 1: "))
qtRacaoG1 = float(input("Quantidade de ração Grupo 1: "))
PsacoG2 = float(input("Peso do saco de ração Grupo 2: "))
qtRacaoG2 = float(input("Quantidade de ração Grupo 2: "))
Transf1 = qtRacaoG1 / 1000
Transf2 = qtRacaoG2 / 1000
Vtotal1 = PsacoG1 - Transf1
Vtotal2 = PsacoG2 - Transf2
print("Restará após uma semana: ", Vtotal1 + Vtotal2, "kg")

Peso do saco de ração Grupo 1: 30
Quantidade de ração Grupo 1: 25000
Peso do saco de ração Grupo 2: 45
Quantidade de ração Grupo 2: 40000
Restará após uma semana:  10.0 kg


QUESTÃO 15

Faça um programa Python para a leitura de duas notas parciais de um aluno. O programa deve calcular a média alcançada por aluno e apresentar: 
A mensagem "Aprovado", se a média alcançada for maior ou igual a sete; 
A mensagem "Reprovado", se a média for menor do que sete; 
A mensagem "Aprovado com Distinção", se a média for igual a dez. 


In [ ]:
nota1 = float(input("Nota 1: "))
nota2 = float(input("Nota 2: "))
media = (nota1 + nota2) / 2
if media >= 7:
    print("Aprovado")
elif media < 7:
    print("Reprovado")
elif media == 10:
    print("Aprovado com Distinção")

Nota 1: 7.6
Nota 2: 9.9
Aprovado


QUESTÃO 16

Faça um Programa Python que leia três números e mostre o maior deles. 

In [ ]:
n1 = int(input("N 1: "))
n2 = int(input("N 2: "))
n3 = int(input("N 3: " ))
if n1 >= n2 >= n3:
  print("O maior número é: ", n1)
elif n2 >= n1 >= n3:
  print("O maior número é: ", n2)
elif n3 >= n1 >= n2:
  print("O maior número é: ", n3)

N 1: 25
N 2: 10
N 3: 49
O maior número é:  49


QUESTÃO 17

Faça um programa Python que pergunte o preço de três produtos e informe qual produto você deve comprar, sabendo que a decisão é sempre pelo mais barato. 

In [ ]:
v1 = float(input('Digite o valor do protudo 1: R$ '))
v2 = float(input('Digite o valor do protudo 2: R$ '))
v3 = float(input('Digite o valor do produto 3: R$ '))

if v1 <= v2 <= v3:
    print("O produto mais barato é o de valor:R$ ", v1)
else:
     if v2 <= v3 <= v1:
        print("O produto mais barato é o de valor:R$ ", v2)
     else:
       if v3 <= v2 <= v1:
            print("O produto mais barato é o de valor: R$", v3)

Digite o valor do protudo 1: R$ 500
Digite o valor do protudo 2: R$ 600
Digite o valor do produto 3: R$ 700
O produto mais barato é o de valor:R$  500.0


QUESTÃO 18

Uma organização resolveu dar um aumento de salário aos seus colaboradores e lhe contrataram para desenvolver o programa que calculará os reajustes. Faça um programa Python que recebe o salário de um colaborador e o reajuste segundo o seguinte critério, baseado no salário atual: 
salários até R$ 280,00 (incluindo) : aumento de 20% 
salários entre R$ 280,00 e R$ 700,00 : aumento de 15% 
salários entre R$ 700,00 e R$ 1500,00 : aumento de 10% 
	salários de R$ 1500,00 em diante : aumento de 5% 
Após o aumento ser realizado, informe na tela: 
a)o salário antes do reajuste; 
b)o percentual de aumento aplicado; 
c)o valor do aumento; 
d)o novo salário, após o aumento. 


In [ ]:
valors = float(input('Qual o valor do seu salário atual: R$ '))

if valors < 280:
  if valors * 1.20:
    print("Salário antes: R$", valors)
    print("Aumento de: 20%.")
    print("Seu aumento é de: R$", (valors * 1.20) - valors)
    print("Seu novo salário é: R$ ", valors * 1.20)
    
if valors >= 280 and valors < 700:
  (valors * 1.15)
  print("Salário antes: R$", valors)
  print("Aumento de: 15%.")
  print("Seu aumento é de: R$", (valors * 1.15) - valors)
  print("Seu novo salário é: R$ ", valors * 1.15)
      
if valors >= 700 and valors < 1500:
  valors * 1.10
  print("Salário antes: R$", valors)  
  print("Aumento de: 10%.") 
  print("Seu aumento é de: R$", (valors * 1.10) - valors)
  print("Seu novo salário é: R$ ", valors * 1.10)

else:
  valors >= 1500
  valors * 1.05
  print("Salário antes: R$", valors)
  print("Aumento de: 5%.")
  print("Seu aumento é de: R$", (valors * 1.05) - valors)
  print("Seu novo salário é: R$ ", valors * 1.05)

Qual o valor do seu salário atual: R$ 1000
Salário antes: R$ 1000.0
Aumento de: 10%.
Seu aumento é de: R$ 100.0
Seu novo salário é: R$  1100.0


QUESTÃO 19

Faça um programa para o cálculo de uma folha de pagamento, sabendo que os descontos são do Imposto de Renda, que depende do salário bruto (conforme tabela abaixo) e 3% para o Sindicato e que o FGTS corresponde a 11% do Salário Bruto, mas não é descontado (é a empresa que deposita). O Salário Líquido corresponde ao Salário Bruto menos os descontos. O programa Python deverá pedir ao usuário o valor da sua hora e a quantidade de horas trabalhadas no mês. 
Desconto do IR: 
Salário Bruto até 900 (inclusive) - isento 
Salário Bruto até 1500 (inclusive) - desconto de 5% 
Salário Bruto até 2500 (inclusive) - desconto de 10% 
Salário Bruto acima de 2500 - desconto de 20% 
Imprima na tela as informações a seguir. No exemplo o valor da hora é 5 e a quantidade de hora é 220. 
Salário Bruto: (5 * 220)        : R$ 1100,00
(-) IR (5%)                     : R$   55,00  
(-) INSS ( 10%)                 : R$  110,00
FGTS (11%)                      : R$  121,00
Total de descontos              : R$  165,00
Salário Liquido                 : R$  935,00


In [14]:
horasTrabalhadas = float(input("Horas trabalhadas: "))
valorPhora = float(input("Valor da hora: "))
salarioBruto = horasTrabalhadas * valorPhora
fgts = salarioBruto * 0.11
inss = salarioBruto * 0.10
print("Salario Bruto: (", valorPhora, "*", horasTrabalhadas, "): ", salarioBruto)
if salarioBruto <= 900:
  print("(-) IR : Isento")
elif salarioBruto > 900 and salarioBruto <= 1500:
  ir = (salarioBruto * 0.05)
  print("(-) IR (5%) : R$" , ir)
elif salarioBruto > 1500 and salarioBruto <= 2500:
  ir = (salarioBruto * 0.10)
  print("(-) IR (10%) : R$" , ir)
else: 
  ir = (salarioBruto * 0.20)
  print("(-) IR (20%) : R$" , ir)
print("(-) INSS (10%) : R$", inss)
print("FGTS (11%) : R$", fgts)
print("Total de descontos : R$", ir + inss)
print("Salário Liquido : R$", salarioBruto - (ir + inss))

Horas trabalhadas: 200
Valor da hora: 15
Salario Bruto: ( 15.0 * 200.0 ):  3000.0
(-) IR (20%) : R$ 600.0
(-) INSS (10%) : R$ 300.0
FGTS (11%) : R$ 330.0
Total de descontos : R$ 900.0
Salário Liquido : R$ 2100.0


QUESTÃO 20

Faça um programa que lê as duas notas parciais obtidas por um aluno numa disciplina ao longo de um semestre, e calcule a sua média. A atribuição de conceitos obedece à tabela abaixo: 
  Média de Aproveitamento  Conceito
  Entre 9.0 e 10.0       A;
  Entre 7.5 e 9.0         B;
  Entre 6.0 e 7.5         C;
  Entre 4.0 e 6.0         D;
  Entre 4.0 e zero       E.
O programa Python deve mostrar na tela as notas, a média, o conceito correspondente e a mensagem “APROVADO” se o conceito for A, B ou C ou “REPROVADO” se o conceito for D ou E. 


In [ ]:
nota1 = float(input("Nota 1: "))
nota2 = float(input("Nota 2: "))
media = (nota1 + nota2) / 2
if  9 < media <= 10 :
  print("A")
  print("Média: ", media)
  print("APROVADO")
elif 7.5 < media <= 9 :
  print("B")
  print("Média: ", media)
  print("APROVADO")
elif 6 < media <= 7.5:
  print("C")
  print("Média: ", media)
  print("APROVADO")
elif 4 < media <= 6:
  print("D")
  print("Média: ", media)
  print("REPROVADO")
elif 4 < media <= 0:
  print("E")
  print("Média: ", media)
  print("REPROVADO")

Nota 1: 4
Nota 2: 5
D
Média:  4.5
REPROVADO


QUESTÃO 21

Faça um Programa Python que peça os 3 lados de um triângulo. O programa deverá informar se os valores podem ser um triângulo. Indique, caso os lados formem um triângulo, se o mesmo é: equilátero, isósceles ou escaleno. 
Dicas: Três lados formam um triângulo quando a soma de quaisquer dois lados for maior que o terceiro; Triângulo Equilátero: três lados iguais; Triângulo isósceles: quaisquer dois lados iguais; Triângulo Escaleno: três lados diferentes; 


In [35]:
lado1 = float(input("Lado 1: "))
lado2 = float(input("Lado 2: "))
lado3 = float(input("Lado 3: "))

if (lado1 + lado2) <= lado3 or (lado1 + lado3) <= lado2 or (lado2 + lado3) <= lado1:
  print("Não é um triângulo")
elif lado1 == lado2 == lado3:
  print('Triângulo equilatero.')
elif lado1 == lado2 != lado3 or lado2 == lado3 != lado1 or lado1 == lado2 != lado3:
  print('Triângulo isósceles.')
elif (lado1 + lado2) > lado3 != (lado1 + lado3) > lado2 != (lado2 + lado3) > lado1:
  print('Triãngulo escaleno')

Lado 1: 2
Lado 2: 4
Lado 3: 3
Triãngulo escaleno


QUESTÃO 22

Faça um Programa Python que peça um número correspondente a um determinado ano e em seguida informe se este ano é ou não bissexto. 

In [8]:
ano = int(input('Ano: '))
if (ano%4==0 and ano%100!=0) or (ano%400==0):
    print('Bissexto')
else:
    print('Não é bissexto')

Ano: 2049
Não é bissexto


QUESTÃO 23

Faça um Programa Python para um caixa eletrônico. O programa deverá perguntar ao usuário a valor do saque e depois informar quantas notas de cada valor serão fornecidas. As notas disponíveis serão as de 1, 5, 10, 50 e 100 reais. O valor mínimo é de 10 reais e o máximo de 600 reais. O programa não deve se preocupar com a quantidade de notas existentes na máquina. 
Exemplo 1: Para sacar a quantia de 256 reais, o programa fornece duas notas de 100, uma nota de 50, uma nota de 5 e uma nota de 1; 
Exemplo 2: Para sacar a quantia de 399 reais, o programa fornece três notas de 100, uma nota de 50, quatro notas de 10, uma nota de 5 e quatro notas de 1. 


In [2]:
numero = int(input('Valor para sacar [10-600]: '))
cem = int(numero / 100)
numero = numero - (cem * 100)
cinquenta = int(numero/50)
numero = numero - (cinquenta * 50)
dez = int(numero/10)
numero = numero - (dez * 10)
cinco = int(numero/5)
numero = numero - (cinco * 5)
um = int(numero/1)
numero = numero - (um * 1)
    
print('Notas R$100,00 = ',cem)
print('Notas R$ 50,00 = ',cinquenta)
print('Notas R$ 10,00 = ',dez)
print('Notas R$  5,00 = ',cinco)
print('Notas R$  1,00 = ',um)


Valor para sacar [10-600]: 549
Notas R$100,00 =  5
Notas R$ 50,00 =  0
Notas R$ 10,00 =  4
Notas R$  5,00 =  1
Notas R$  1,00 =  4


QUESTÃO 24

Uma fruteira está vendendo frutas com a seguinte tabela de preços: 
Até 5 Kg           Acima de 5 Kg 
Morango         R$ 2,50 por Kg          R$ 2,20 por Kg
Maçã            R$ 1,80 por Kg          R$ 1,50 por Kg
Se o cliente comprar mais de 8 Kg em frutas ou o valor total da compra ultrapassar R$ 25,00, receberá ainda um desconto de 10% sobre este total. Escreva um programa Python para ler a quantidade (em Kg) de morangos e a quantidade (em Kg) de maças adquiridas e escreva o valor a ser pago pelo cliente.


In [40]:
qMorango = float(input("Quantidade morango em Kg: "))
qMaca = float(input("Quantidade maçã em Kg: "))
valorKg = qMorango + qMaca
precoM1 = qMorango * 2.50
precoM2 = qMorango * 2.20
precoMa1 = qMaca * 1.80
precoMa2 = qMaca * 1.50
valorTotal = (precoM1 or precoM2) + (precoMa1 or precoMa2)

print("Valor total: R$ ", valorTotal)
if qMorango <= 5:
  precoM1 = qMorango * 2.50
  print("Preço morango: R$ ", precoM1)
else:
  qMorango > 5
  precoM2 = qMorango * 2.20
  print("Preço morango: R$ ", precoM2)
if qMaca <= 5:
  precoMa1 = qMaca * 1.80
  print("Preço maça: R$ ", precoMa1)
else:
  qMaca > 5
  precoMa2 = qMaca * 1.50
  print("Preço maça: R$ ", precoMa2)
if valorKg > 8 or valorTotal > 25:
  print("Valor com desconto: R$ ", valorTotal - (valorTotal * 0.10))

Quantidade morango em Kg: 5
Quantidade maçã em Kg: 5
Valor total: R$  21.5
Preço morango: R$  12.5
Preço maça: R$  9.0
Valor com desconto: R$  19.35


QUESTÃO 25

Elabore um programa Python que dada a idade de um nadador classifica-o em uma das seguintes categorias:  infantil A = 5 - 7 anos; infantil B = 8-10 anos; juvenil A = 11-13 anos; juvenil B = 14-17 anos; adulto = maiores de 18 anos.

In [ ]:
idadeN = int(input("Idade do nadador: "))
if 5 <= idadeN <= 7:
  print("Infantil A")
elif 8 <= idadeN <= 10:
  print("Infantil B")
elif 11 <= idadeN <= 13:
  print("Juvenil A")
elif 14 <= idadeN <= 17:
  print("Juvenil B")
elif idadeN >= 18:
  print("Adulto")

Idade do nadador: 24
Adulto


QUESTÃO 26

Tendo como dados de entrada a altura e o sexo de uma pessoa, construa um programa Python que calcule seu peso ideal, utilizando as seguintes fórmulas: para homens: (72.7*h) – 58 e para mulheres: (62.1*h) - 44.7 (h = altura).

In [37]:
h = float(input('Qual a sua altura? '))
s = int(input('Digite 1 para feminino ou 2 para masculino: '))
if s > 3:
  print('Digite apenas 1 ou 2 na segunda pergunta. ')
idealm = (62.1 * h) - 44.7
idealh = (72.7 * h) - 58

if s == 1:
  idealm
  print('Seu peso ideal é: ', idealm)
else:
  s == 2
  idealh
  print('Seu peso ideal é: ', idealh)

Qual a sua altura? 1.60
Digite 1 para feminino ou 2 para masculino: 2
Seu peso ideal é:  58.32000000000001


QUESTÃO 27

O IPVA é calculado de acordo com o tipo de veículo, cuja alíquota é definida por cada Estado sobre o valor venal do carro de acordo com a Tabela Fipe (Fundação Instituto de Pesquisas Econômicas), variando entre 1% e 4%. Escreva um programa em Python para calcular o valor do IPVA de um veículo com base nas informações a seguir:

Alíquota
Tipo do Veículo
4,0%
Automóveis
3,0%
Caminhonetes de carga e furgão.
2,0%
Automóveis para transporte público (ex: táxi, escolar) 
2,0%
Motocicletas.
1,0%
Veículos de locadoras.
1,0%
Ônibus, micro-ônibus, caminhão, caminhão trator.


OBS: o valor venal do carro deverá ser fornecido pelo usuário.


In [6]:
valor = float(input('Indique o valor do seu carro: R$ '))
tipo = float(input('Digite 1 se seu transporte é automóvel, 2 para caminhonetes, 3 para transporte público, 4 para motocicleta, 5 para veículos de locadora, 6 para caminhão: '))
if 0 <= tipo > 5:
  print('Digite apenas números de 1 a 5 na segunda pergunta. ')
valor1 = valor - (valor * 0.96)  # (1 - 0.04)
valor2 = valor - (valor * 0.97) # (1 - 0.03)
valor3 = valor - (valor * 0.98) # (1 - 0.02)
valor4 = valor - (valor * 0.98) 
valor5 = valor - (valor * 0.99) # (1 - 0.01)
valor6 = valor - (valor * 0.99)

if tipo == 1:
  valor1
  print('Seu IPVA é: R$ ', valor1)
if tipo == 2:
  valor2
  print('Seu IPVA é: R$ ', valor2)
if tipo == 3:
   valor3
   print('Seu IPVA é: R$ ', valor3)
if tipo == 4:
   valor4
   print('Seu IPVA é: R$ ', valor4)
if tipo == 5:
   valor5
   print('Seu IPVA é: R$ ', valor5)
if tipo == 6:
   valor6
   print('Seu IPVA é: R$ ', valor6)

Indique o valor do seu carro: R$ 90000
Digite 1 se seu transporte é automóvel, 2 para caminhonetes, 3 para transporte público, 4 para motocicleta, 5 para veículos de locadora, 6 para caminhão: 1
Seu IPVA é: R$  3600.0


QUESTÃO 28

Escreva um programa em Python para ler a capacidade de um elevador (em toneladas) e o peso total das pessoas em seu interior (em quilogramas). Informar se o elevador está liberado para subir ou se excedeu a carga máxima.

In [7]:
pessoa1 = float(input('Digite seu peso: kg '))
pessoa2 = float(input('Digite seu peso: kg '))
pessoa3 = float(input('Digite seu peso: kg '))
pessoa4 = float(input('Digite seu peso: kg '))
pessoa5 = float(input('Digite seu peso: kg '))
pessoa6 = float(input('Digite seu peso: kg '))
pesototal = pessoa1 + pessoa2 + pessoa3 + pessoa4 + pessoa5 + pessoa6

pesotonelada = pesototal / 1000

if pesotonelada > 0.5:
  print('Excedeu a carga máxima. Valor mínimo: 0,5 T. Valor da carga: ', pesotonelada, 'T')
else:
    pesotonelada < 0.5
    print('Elevador liberado. Valor mínimo: 0,5 T. Valor da carga: ', pesotonelada, 'T')

Digite seu peso: kg 80
Digite seu peso: kg 96
Digite seu peso: kg 75
Digite seu peso: kg 85
Digite seu peso: kg 67
Digite seu peso: kg 59
Elevador liberado. Valor mínimo: 0,5 T. Valor da carga:  0.462 T


QUESTÃO 29

Desenvolver um programa em Python para calcular a conta de água para a CAGEPA. O custo da água varia dependendo se o consumidor é residencial, comercial ou industrial. A regra para calcular a conta é:

– Residencial: R$50,00 de taxa mais R$0,05 por m³ gastos;
– Comercial: R$500,00 para os primeiros 80 m³ gastos mais R$0,25 por m³ gastos acima dos 80 m³;
– Industrial: R$800,00 para os primeiros 100 m³ gastos mais R$0,04 por m³ gastos acima dos 100 m³;

O programa deverá ler o número da conta do cliente, consumo de água por metros cúbicos e o tipo de consumidor (residencial, comercial e industrial). Como resultado, imprima o número da conta do cliente e o valor real a ser pago pelo mesmo.


In [ ]:
numC = float(input("Número da conta do cliente: "))
consumo = float(input("Consumo de água / m3: "))
tipoConsumidor = str(input("Tipo de consumidor: "))
consumoR = 
if 

QUESTÃO 30

Fazer um programa em Python para determinar o quadrante ao qual pertence um determinado ponto P(x, y).


In [5]:
x = float(input('Digite o valor de X: '))
y = float(input('Digite o valor de y: '))

if 0 == x:
  print('P pertence ao eixo Y.')
else:
  0 == y
  print('P pertence ao eixo X.')

if 0 < x and y > 0:
  print('P pertence ao 1° quadrante.')
elif 0 > x and y > 0:
  print('P pertence ao 2° quadrante.')
elif 0 >x and 0 > y:
  print('P pertence ao 3° quadrante.')
elif 0 < x and y < 0:
  print('P pertence ao 4° quadrante.')

Digite o valor de X: -5
Digite o valor de y: 3
P pertence ao eixo X.
P pertence ao 2° quadrante.
